dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -83,11 +94,21 @@ private static String generate18(String projectName, double[] percentages, doubl
             }
         }
         for (int i = 0; i + 1 < values.length; i += 2) {
-            double val = values[i] - values[i + 1];
+            double val;
+            if (!nonZero[i]) {
+                val = values[i] - values[i + 1];
+                // System.out.println("..." + projectName + i + " " + val);
```

Comment:
Instead of removing these comments you can also add a debug boolean flag as a program argument such that when that flag is given to the program then these printlns will execute. Otherwise they won't print.